test/unit/spec/common/appState-Spec.js

```diff
@@ -240,6 +241,9 @@ define(['common/fsm'], (Fsm) => {
             }
             expect(alive).toBeTruthy();
         });
+        afterEach(() => {
+            TestUtil.clearRuntime();
+        });
```

Comment:
hmmm, who could be responsible for the comment cruft below these lines?!

Reply:
I WONDER.